src/libc/ishr.src

```diff
@@ -1,12 +1,10 @@
 	assume	adl=1
```

Comment:
I just noticed: What is this doing in _libc_ instead of _crt_ like everything else?